auto_editor/AddCommentsRule.py

```diff
@@ -113,5 +113,53 @@ def run_rule(self, project: StructuredProjectSource,
                     w_type = "DPCT"
                     return project
 
+                if w_type == "DPCT1009":
+                    del all_items[warning_first_line: warning_last_line + 1]
+                    comment_item = LineItem(prefix + "/*\n"
+                                                     + prefix + "CTA1009:" + count + ": CTA1009:count number: SYCL uses exceptions to report errors and does not use the error codes. \n"
+                                                     + prefix + "The original code was commented out and a warning string was inserted. You need to rewrite this code\n"
+                                                     + prefix + "*/\n")
+                    all_items.insert(warning_first_line, comment_item)
+                    w_type = "DPCT"
+                    return project
+
+                if w_type == "DPCT1010":
+                    del all_items[warning_first_line: warning_last_line + 1]
+                    comment_item = LineItem(prefix + "/*\n"
+                                                     + prefix + "CTA1010:" + count + ": Base on the experience, strongly recommended to ignore this warning.\n"
+                                                     + prefix + "*/\n")
+                    all_items.insert(warning_first_line, comment_item)
+                    w_type = "DPCT"
+                    return project
+
+                if w_type == "DPCT1011":
+                    del all_items[warning_first_line: warning_last_line + 1]
+                    comment_item = LineItem(prefix + "/*\n"
+                                                     + prefix + "CTA1011:" + count + ": Delete the namespace.\n"
+                                                     + prefix + "*/\n")
+                    all_items.insert(warning_first_line, comment_item)
+                    w_type = "DPCT"
+                    return project
+
+                if w_type == "DPCT1007":
+                    del all_items[warning_first_line: warning_last_line + 1]
+                    comment_item = LineItem(prefix + "/*\n"
+                                                     + prefix + "CTA1007:" + count + ": Migration of this CUDA API is not supported by the Intel(R)DPC Compatibility Tool. Please rewrite it by yourself.\n"
```

Comment:
This is the exact same message as DPCT1007. I don't think that's a good idea, as our tool adds no value. Please also check that this is not the case for any of the other recommendations. Obviously, we should only make a recommendation if we have some information to provide. At least, we should provide info about in how many of x cases manual changes had to be made

Reply:
sure, so maybe delete 1007  recommendation